mm2src/mm2_test_helpers/dummy_files/iris_test_history.json

```diff
@@ -0,0 +1,170 @@
+        "tx_hash": "6BA67D4C7A0F48E5D85EE18EF5A649ED456B539D902828D56849B57DCFF5914B",
+        "from": [
+            "iaa1z609g9z8ef4jlkcfgg36c2ljgcmna50l2yjra0"
+        ],
+        "to": [
+            "iaa1erfnkjsmalkwtvj44qnfr2drfzdt4n9ldh0kjv"
+        ],
+        "total_amount": "0.026427",
+        "spent_by_me": "0.026427",
+        "received_by_me": "0",
+        "my_balance_change": "-0.026427",
+        "block_height": 6148319,
+        "timestamp": 0,
+        "fee_details": {
+            "type": "Utxo",
+            "coin": "IRIS-TEST",
+            "amount": "0.026426"
+        },
+        "coin": "IRIS-TEST",
+        "internal_id": "4538314545353844354538344630413743344437364142360000000000000000",
+        "transaction_type": "StandardTransfer"
+    },
+    {
+        "tx_hex": "0a2a696161317a36303967397a386566346a6c6b63666767333663326c6a67636d6e6135306c32796a726130122a696161316572666e6b6a736d616c6b7774766a3434716e6672326472667a6474346e396c6468306b6a762a0a0a05756e79616e12013132403834373331663565393739643965616263343838393234356131613165663932396430386437383832353163353133633834303632373535313032343765326540e807",
+        "tx_hash": "F1C633D16C1578A16450555A06349989A4B1E2223D7AAA2FE12396A5F855FD9B",
+        "from": [
+            "iaa1z609g9z8ef4jlkcfgg36c2ljgcmna50l2yjra0"
+        ],
+        "to": [
+            "iaa1erfnkjsmalkwtvj44qnfr2drfzdt4n9ldh0kjv"
+        ],
+        "total_amount": "0.028456",
+        "spent_by_me": "0.028456",
+        "received_by_me": "0",
+        "my_balance_change": "-0.028456",
+        "block_height": 6148318,
+        "timestamp": 0,
+        "fee_details": {
+            "type": "Utxo",
+            "coin": "IRIS-TEST",
+            "amount": "0.028455"
+        },
+        "coin": "IRIS-TEST",
+        "internal_id": "4135353530353436314138373531433631443333364331460000000000000000",
+        "transaction_type": "StandardTransfer"
+    },
+    {
+        "tx_hex": "0a2a696161317a36303967397a386566346a6c6b63666767333663326c6a67636d6e6135306c32796a7261301240304544424136304632423642383739324133333739423232453445323144373243343145413530464436353331433243383445413135353937303834373432391a4034353138383334373164316536383461323334663362323063626638646137393338636330643038383562626237326266303837363465386431356536353138",
+        "tx_hash": "10D92256317083E49F06A9DD2184714DEEE9C55A505CFEF037BE8E87E9FB82ED",
+        "from": [
+            "iaa1z609g9z8ef4jlkcfgg36c2ljgcmna50l2yjra0"
+        ],
+        "to": [
+            "iaa1z609g9z8ef4jlkcfgg36c2ljgcmna50l2yjra0"
+        ],
+        "total_amount": "0.01",
+        "spent_by_me": "0",
+        "received_by_me": "0.01",
+        "my_balance_change": "0.01",
```

Comment:
`iaa1z609g9z8ef4jlkcfgg36c2ljgcmna50l2yjra0` paid fee during this tx, so resulting `my_balance_change` should be `0.01 - 0.02648`.